fetchQuotes.js

```diff
@@ -0,0 +1,13 @@
+import fetch from 'node-fetch';
+
+const fetchQuotes = async () => {
+  const url = 'https://futuramaapi.herokuapp.com/api/characters/Bender';
```

Comment:
This will return Bender quotes, but if you leave off that part of the url it will be random across all characters